daemon/execdriver/lxc/lxc_template.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/opencontainers/runc/libcontainer/label"
 )
 
+// LxcTemplate is the default template for lxc
```

Comment:
It is the 1 and only template, no?

Also should mention this is used to configure LXC.